### ISSUE TITLE
Cannot change allready entered modules

### ISSUE THREAD

**Issue by Rignchen** [jobtrek/grades-view-solid#17]
If you enter a module's name with a grade and submit it, then you enter the module's number back and try submitting it again it will say that the module doesn't exist instead of changing the module's grade

**Comment by fuzoh** [comment]
Right, the autocomplete allows only  modules that are not already taken.
